README.md

```diff
@@ -40,3 +40,74 @@ if ($file->getSize()) {
 // Access a resource provided by this Divide.IQ server.
 $result = $divideIq->request('stockbase_stock');
 ```
+
+Example Usage orderRequest POST
+=============
+```php
+use DivideBV\PHPDivideIQ\DivideIQ;
+use DivideBV\PHPDivideIQ\Models;
+ 
+
+$person = new Models\Person(); // Requires surname of customer.
+$address = new Models\Address(); // Requires Street, StreetNumber, Zipcode, City and CountryCode in ISO3 format. (NLD)
```

Comment:
Indeed, added to readme in commit https://github.com/Nav-Appaiya/PHPDivideIQ/commit/5b6409dcd6219fab703baa7522cf81abc05df2db